doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+* :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` Solves the reduced system of equations.
+* :func:`KrylovSolver.kryexpand() <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` Expands the Krylov subspace.
+
+.. warning:: These methods must be run consecutively.
+
+Usage
+=====
+
+Generate the full file (``.full``) for Krylov method using Mechanical APDL:
+
+.. code:: py
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    
+    >>> mapdl = launch_mapdl()
+    >>> mapdl.prep7()
+
+	# Generate the FEA model (mesh, constraints, loads, etc.)
+  # ...
+
+    >>> mapdl.run("/SOLU")
+    >>> mapdl.antype("HARMIC")  # HARMONIC ANALYSIS
+    >>> mapdl.hropt("KRYLOV")
+    >>> mapdl.eqslv("SPARSE")
+    >>> mapdl.harfrq(0,1000)  # Set beginning and ending frequency
+    >>> mapdl.nsubst(100)  # Set the number of frequency increments
+    >>> mapdl.wrfull(1)  # GENERATE .FULL FILE AND STOP
+    >>> mapdl.solve()
+    >>> mapdl.finish()
+
+Create Instance of the Krylov Class
+-----------------------------------
+
+.. code:: py
+    
+    >>> mk = mapdl.krylov
+
+Call the :func:`krygensub <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` method which creates the Krylov subspace:
+Build a subspace of Size / Dimension 10 and at a frequency of 500 Hz.
+
+.. code:: py
+
+    >>> Qz = mk.krygensub(10, 500, True, True)
+
+Return the Krylov subspace
+--------------------------
+
+Call the :func:`krysolve <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` method which reduces system of equations and solve at each frequency:
+Solve from 0 Hz to 1000 Hz with 100 intervals in between, with stepped loading   
+
+.. code:: py
+
+    >>> Yz = mk.krysolve(0, 1000, 100, 1, True)
+
+
+Return the Reduced solution over Frequency range
+------------------------------------------------
+            
+Call the :func:`kryexpand <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` method which expand the reduced solution back to FE space:
+Output the expanded solution and calculate residual.   
+
+.. code:: py
+
+    >>> res = mk.kryexpand(True, 3)
+
+Return Ndarray (if out_key = True) solution vectors mapped to User order.
+
+.. note:: The Ndarray returned by the method ``kryexpand`` contains the node number along with dof solution,
+          for each of the calculated frequency.
+
+Get the dof solution at a specific frequency
+--------------------------------------------
+
+.. code:: py
+
+   # Get the nodal solution at freq number 3``````
+   >>> node_order = res[2]['node'] # Get the nodal order   
+   >>> for node_num in node_order:
+   >>> 	   nodal_sol = res[2][node_num]['x'] # Get the nodal solution for each node
+
+Example
+=======
+
+Examples of using the krylov PyMAPDL are available in :ref:`krylov_example`.
+
+Requirements
+============
+
+To use ``krylov`` feature, you need to meet the following requirements:
+
+* ANSYS MAPDL version should be 2022R2 or newer.
```

Comment:
```suggestion
To use the Kyylov method in PyMAPDL, these requirements must be met:

* Ansys MAPDL version 2022 R2 or later
```

Reply:
Are other requirements to be added? A list should only be used if there are two or more entries.